sdk/ai/azure-ai-projects/azure/ai/projects/aio/operations/_operations.py

```diff
@@ -5401,6 +5401,46 @@ def __init__(self, *args, **kwargs) -> None:
         self._serialize: Serializer = input_args.pop(0) if input_args else kwargs.pop("serializer")
         self._deserialize: Deserializer = input_args.pop(0) if input_args else kwargs.pop("deserializer")
 
+    @distributed_trace_async
+    async def create_application_insights_if_not_exists(self, resource_group_name: str, resource_name: str, location: str) -> str:
```

Comment:
Thanks! I'll try the implementation of this function shortly. At first lance, this is the operations for async calls, while this function seems to be sync?